docs/LICENSE

```diff
@@ -0,0 +1,395 @@
+Attribution 4.0 International
```

Comment:
Why this license is in the docs folder? Which license is the right one? Please clarify that.

Reply:
removd